api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1066,6 +1085,9 @@ message Cluster {
 
     // Optional configuration for the RoundRobin load balancing policy.
     RoundRobinLbConfig round_robin_lb_config = 56;
+
+    // Optional configuration for the Deterministic Aperture load balancing policy.
+    DeterministicApertureLbConfig deterministic_aperture_lb_config = 57;
```

Comment:
All these API should be added as load balancing policy extension configuration.

Reply:
Thanks for looking @wbpcode . The existing ones (least request, maglev etc) seem to be present at both places (`api/envoy/lextensions/oad_balancing_policies/..` as well as in `api/envoy/config/cluster/v3/cluster.proto`). Is that the intent for this also? If so, how do I include `RingHash` message from `api/envoy/lextensions/oad_balancing_policies/deterministic_aperture.proto`? If I do, then in the code, I will have to pass the extension API message to `https://github.com/envoyproxy/envoy/blob/main/source/common/upstream/ring_hash_lb.h#L46` which currently does not accept the extension API.

Reply:
ping @wbpcode

Reply:
> Is that the intent for this also?
It' just for back-compatibility.

Reply:
> which currently does not accept the extension API.

Some minor refactors may be necessary to construct `RingHashLoadBalancer` from the extension API.

Reply:
@wbpcode I am updating the PR to use the extension API. I have added a conversion method from  extension RingHash configuration proto to cluster RingHash proto to localize the change.